app/views/staff/session_types/_session_type.html.haml

```diff
@@ -0,0 +1,21 @@
+%tr{ id: "session_type_#{session_type.id}" }
+  %td= session_type.name
+  %td= truncate(session_type.description, length: 60)
+  %td= session_type.duration
+  %td= session_type.public? ? "\u2713" : ""
+  - unless !current_user.organizer_for_event?(event)
+    %td.actions
+      = link_to "Edit", "#", class: "btn btn-primary btn-xs",
```

Comment:
After appending a new session type to the dom, the remove button works, but the edit button does not work unless a crud method is specified or page is refreshed, and I can't figure out the best way to do that without redirect. Would love feedback. Same issue holds true for tracks.